test/gtest/uct/test_amo_fadd_fxor.cc

```diff
@@ -11,14 +11,14 @@ class uct_amo_fadd_fxor_test : public uct_amo_test {
 public:
 
     template <typename T, uct_atomic_op_t OP>
-    void test_fop(T (*op)(T, T)) {
+    void test_fop(T (*op)(T, T), ucs_memory_type mem_type = UCS_MEMORY_TYPE_HOST) {
```

Comment:
seems test_fop was changed but it's not really called with non-default mem_type argument,
maybe test_fop can run all supported memory types?